frontend/web/components/metadata/MetadataPage.tsx

```diff
@@ -0,0 +1,351 @@
+                                fill='#20c997'
+                              />
+                            </>
+                          )}
+                        </>
+                      ) : (
+                        <>
+                          <Icon name='close-circle' width={20} />
+                        </>
+                      )}
+                    </>
+                  ) : (
+                    <>
+                      <Icon name='close-circle' width={20} />
+                    </>
+                  )}
+                </div>
+              </div>
+              <div className='table-column' style={{ width: '150px' }}>
+                <div className='table-column' style={{ width: '150px' }}>
+                  {metadata.content_type_fields.find(
+                    (m) => m.content_type === segmentContentType?.id,
+                  ) ? (
+                    <>
+                      {metadata.content_type_fields.some(
+                        (field) =>
+                          field.content_type === segmentContentType?.id,
+                      ) ? (
+                        <>
+                          {metadata.content_type_fields.some(
+                            (field) =>
+                              field.content_type === segmentContentType?.id &&
+                              field.is_required_for &&
+                              field.is_required_for.length > 0,
+                          ) ? (
+                            <>
+                              <Icon name='required' width={20} />
+                            </>
+                          ) : (
+                            <>
+                              <Icon
+                                name='checkmark-circle'
+                                width={20}
+                                fill='#20c997'
+                              />
+                            </>
+                          )}
+                        </>
+                      ) : (
+                        <>
+                          <Icon name='close-circle' width={20} />
+                        </>
+                      )}
+                    </>
+                  ) : (
+                    <>
+                      <Icon name='close-circle' width={20} />
+                    </>
+                  )}
+                </div>
+              </div>
+              <div
+                className='table-column'
+                style={{ width: metadataWidth[2] }}
+                onClick={() => {
+                  editMetadata(`${metadata.id}`, metadata.content_type_fields)
+                }}
+              >
+                <div
+                  className='table-column'
+                  style={{ width: '150px' }}
+                  onClick={() => {
+                    editMetadata(`${metadata.id}`, metadata.content_type_fields)
+                  }}
+                >
+                  {metadata.content_type_fields.find(
```

Comment:
Same as above

Reply:
With the new changes in the UI, this code will no longer exist.